package/MDAnalysis/core/_get_readers.py

```diff
@@ -94,8 +93,6 @@ def get_reader_for(filename, format=None):
         elif isinstance(filename, mmtf.MMTFDecoder):
             # mmtf slurps mmtf object
             format = 'MMTF'
-        elif isinstance(filename, pmd.Structure):
```

Comment:
Removing this breaks the auto detection of parmed objects right? Is there a hack you can do like if str(obj.__class__.split(‘.’)[0]) == “parmed” which lets you not have to import it?